src/utils.js

```diff
@@ -38,6 +38,8 @@ export function expressMiddleware (monitor, name) {
           monitor.measure(k, d[0] * 1000 + d[1] / 1000000);
           monitor.count(k);
         }
+        monitor.measure(['all', stat], d[0] * 1000 + d[1] / 1000000);
```

Comment:
Ah, that might be right...
Actually, I'm little suspicious as to the utility of all these alls.. some of them sure...

But given that 99% is success, then we might not need both all and success. As the percentiles will be the same...